CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -164,16 +164,16 @@ def getOutputCell(cell, wsOut):
     return wsOut[cell.coordinate]
 
 
-def getNonLinguisticCharacterCount(val, additionalCharactersToCatch=None, characterList=None):
+def getNonLinguisticCharacterCount(val, characterList=NON_LINGUISTIC_CHARACTERS, additionalCharactersToCatch=""):
```

Comment:
Looks like this was a bug, the places where it is called that is the order already.